packages/api/tests/endpoints/test-rules.js

```diff
@@ -449,8 +656,62 @@ test('PUT returns 400 for name mismatch between params and payload',
     t.falsy(record);
   });
 
+test('PUT replaces onetime rule for rerun', async (t) => {
```

Comment:
I added the test because the following line didn't seem to ever be explicitly tested: 
https://github.com/nasa/cumulus/blob/master/packages/api/endpoints/rules.js#L116

I could be wrong but I didn't find a test for that.

Reply:
It's fine to add that test, but all we're testing then is that the original rule passed in is returned by the endpoint. It's a fine test to write, but we can really simplify the test logic in that case. And update the test name.

```suggestion
test('PUT returns original rule for rerun', async (t) => {
```